packages/cashier/src/App.jsx

```diff
@@ -1,27 +1,20 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
@vinu-deriv believe that we should follow our js [guideline](https://github.com/george-usynin-deriv/deriv-app/blob/master/docs/JavaScript/README.md#import-rules), and i think it's better to leave only React as we do in all our components:
`import React from 'react';`
and then use: 
`React.useEffect()`